locale_update.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
+
+##
+# This script is designed to update the l10n files and locale registration
+# for the standalone loop client. The source of the localization files
+# is https://github.com/mozilla/loop-client-l10n repository.
+# The loop client repo is assumed to be https://github.com/mozilla/loop-client.
+#
+# Run this script from the local version of loop-client. It assumes that a local
+# version of loop-client-l10n is in a parallel directory: ../loop-client-l10n.
+# Changes are then pushed back to loop-client.
+##
+
+import sys
+import os
+import shutil
+import io
+import re
+
+l10n_src = "../loop-client-l10n/l10n/"
+content_dir = "content/"
+index_file_name = "index.html"
+
+def createLocale(dst_name_list, src_dir):
+    if len(src_dir) < 1 or src_dir[0] == "." or src_dir == 'templates':
+        return dst_name_list
+    
+    dst_dir = src_dir.replace('_', '-')
+    shutil.copytree(l10n_src + src_dir, content_dir + "l10n/" + dst_dir)
+    dst_name_list.append(dst_dir)
+    return dst_name_list
+
+def main():
+    print "deleting existing l10n content tree"
+    shutil.rmtree(content_dir + "l10n", True)
```

Comment:
in general, we try not to concat directly paths like that, since that's not compatible with how windows work. Please use [`os.path.join`](https://docs.python.org/2/library/os.path.html#os.path.join) instead.